clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -646,14 +661,50 @@ check_duplicate_syscall_with_same_pc()
     return true;
 }
 
+bool
+check_rseq_side_exit_discontinuity()
+{
+    // Negative test: Seemingly missing instructions in a basic block due to rseq side
+    // exit.
+    instr_t *store = XINST_CREATE_store(GLOBAL_DCONTEXT, OPND_CREATE_MEMPTR(REG2, 0),
+                                        opnd_create_reg(REG1));
+    instr_t *move1 =
+        XINST_CREATE_move(GLOBAL_DCONTEXT, opnd_create_reg(REG1), opnd_create_reg(REG2));
+    instr_t *move2 =
+        XINST_CREATE_move(GLOBAL_DCONTEXT, opnd_create_reg(REG1), opnd_create_reg(REG2));
+    instr_t *jmp =
```

Comment:
nit: since the rseq test cares about whether the jump is conditional, it will be clearer to name this s/jmp/cond_jmp/